config/ftbquests/quests/chapters/bigger_reactors.snbt

```diff
@@ -0,0 +1,1084 @@
+				""
+				"Note: To complete this quest, you'll need to create a &dTask Screen&r of any size. Once placed, you can right-click the screen and select this quest as the requirement, then output power into the task screen block to fill it up and complete the quest."
+				""
+				"{image:atm:textures/questpics/biggerreactors/reactor_turbine_example.png width:300 height:150 align:1}"
+			]
+			dependencies: [
+				"3FC7FDAF84871963"
+				"67AFCBCE7AAC3089"
+				"186731580B14F9D2"
+				"775D176081DD75F5"
+			]
+			size: 2.0d
+			min_width: 300
+			id: "4ED36AA3766E842B"
+			tasks: [
+				{
+					id: "4A94CC9DA47A370C"
+					type: "item"
+					title: "Quest Task Screens"
+					item: {
+						id: "itemfilters:or"
+						Count: 1b
+						tag: {
+							items: [
+								{
+									id: "ftbquests:screen_1"
+									Count: 1b
+								}
+								{
+									id: "ftbquests:screen_3"
+									Count: 1b
+								}
+								{
+									id: "ftbquests:screen_5"
+									Count: 1b
+								}
+								{
+									id: "ftbquests:screen_7"
+									Count: 1b
+								}
+							]
+						}
+					}
+				}
+				{
+					id: "24146672439051F5"
+					type: "forge_energy"
+					value: 10000000L
+					max_input: 100000L
+				}
+			]
+			rewards: [
+				{
+					id: "5AE542B84586D0BA"
+					type: "xp"
+					xp: 1000
+				}
+				{
+					id: "6612F39435CC428F"
+					type: "random"
+					exclude_from_claim_all: true
+					table_id: 5564196992594175882L
+				}
+			]
+		}
+		{
+			title: "Turbine Construction"
+			x: -2.0d
+			y: 10.0d
+			shape: "diamond"
+			description: [
+				"The &9Turbine&r has a minimum shape of 5x4x5, but can go up to 32x192x32 in size."
+				""
+				"Just like the Reactor, the frame must be built out of &dCasings&r, but the walls can be &9Turbine Glass&r instead."
+			]
+			dependencies: ["4415C9F8DA2D7E68"]
+			id: "2D592669F4D41793"
+			tasks: [{
+				id: "2FC2A5929C2E0EA9"
+				type: "item"
+				item: "biggerreactors:turbine_glass"
+				count: 4L
+			}]
+			rewards: [
+				{
+					id: "01C09E75947CCBF5"
+					type: "item"
+					item: "biggerreactors:turbine_glass"
+					count: 2
+					random_bonus: 2
+				}
+				{
+					id: "46A906727C46868C"
+					type: "xp"
+					xp: 10
+				}
+			]
+		}
+		{
+			x: -3.0d
+			y: 10.0d
+			description: [
+				"Right-clicking on a Turbine Terminal when it is fully built will show you the Turbine Interface."
+				""
+				"Here, you can see all of the stats for the Turbine. Hovering over each will tell you more info about each one."
+				""
+				"On the right side, you'll have 2 arrows to control the &9Input Flow Rate&r. This controls how much heated fluid is pumped into the Turbine. To know how much you should set this to, check your reactor's &dExhaust Flow Rate&r as a starting point. "
```

Comment:
Turbines automatically set themselves to the optimal flow rate for 1800RPM operation given their blade setup. This is documented exactly nowhere. This is also only done once per turbine, so, potentially minimally useful.

Reply:
So during the first setup, it automatically sets itself to the optimal flow rate? Is there a way to reset it to do it again?

Reply:
no, but there should be.

Reply:
Alright, I'll leave it out for now.